packages/dart/noports_core/lib/src/sshnp/util/srvd_channel/srvd_channel.dart

```diff
@@ -78,6 +78,7 @@ abstract class SrvdChannel<T> with AsyncInitialization, AtClientBindings {
     } else {
       _host = params.host;
       _portA = params.port;
+      _portB = params.port;
```

Comment:
Our abstractions aren't quite right here; _portB is irrelevant when we are not using the srvd; but for safety's sake we need to set it to something